Resources/input/fsh/profiles/3.1_Prescription_ServiceRequest.fsh

```diff
@@ -0,0 +1,135 @@
+
+* identifier ^slicing.discriminator.type = #pattern
+* identifier ^slicing.discriminator.path = "system"
+* identifier ^slicing.rules = #open
+* identifier ^slicing.description = "Defines identifiers that can be used for this context"
+
+* identifier 1..* MS
+* identifier contains requestId 1..1
+* identifier[requestId] only GEM_ERP_MEDREQ_PR_RequestIdentifier
+
+* basedOn 1..1 MS
+  * ^short = "MedicationRequest, der von der Pflegeeinrichtung gewünscht wird oder vom Arzt erfüllt wurde."
+  * ^comment = "Einem ServiceRequest ist genau ein MedicationRequest zugeordnet, sodass unabhängige Bearbeitungen möglich sind."
+* basedOn only Reference(GEM_PR_ERP_MEDREQ_MedicationRequest or $KBV_PR_ERP_Prescription)
+
+* replaces 0..1 MS
+* replaces only Reference(GEM_PR_ERP_MEDREQ_Prescription_ServiceRequest)
+  * ^short = "Wenn eine Pflegeeinrichtung ihre alte Anfrage stornieren möchte und eine neue Stellt, kann hier das zu stornierende ServiceRequest angegeben werden."
+  * ^comment = "Nur vom Requester zu nutzen, um eine alte Anfrage zu stornieren"
+
+* requisition 1..1 MS
+* requisition only GEM_ERP_MEDREQ_PR_RequestGroupIdentifier
+  * ^short = "Identifier des Vorgangs. Alle ServiceRequests innerhalb eines Vorgangs erhalten die gleiche ID."
+  * ^comment = "Ist als Fall oder VorgangsID zu verstehen, um nachverfolgen zu können zu welcher Anfrage der ServiceRequest gehört."
+
+* status MS
+//* status obeys sr-1
+* status from GEM_VS_MEDREQ_RequestStatus
+  * ^short = "Gibt den Bearbeitungsstand eines ServiceRequests an"
+
+* intent = #order (exactly)
+
+* code 1..1 MS
+* code from GEM_VS_MEDREQ_Service_Request_Code
+  * ^short = "Gibt die Art des ServiceRequests an."
+  * ^comment = "#prescription-request dient der Anfrage eines Rezeptes an einen Arzt. #pharmacy-delivery-request dient als ServiceRequest für eine Apotheke zur Belieferung"
+
+* subject MS
+* subject only Reference($KBV_PR_FOR_Patient)
+
+* orderDetail 1..1 MS
+* orderDetail from GEM_VS_MEDREQ_Prescription_Fullfillment_Type (required)
+  * ^short = "Gibt an wie die Rezeptanfrage erfüllt werden soll."
+  * ^comment = "Wenn das Rezept an eine Apotheke geht, muss ein entsprechender ServiceRequest im Bundle enthalten sein."
+
+* occurrence[x] 1..1 MS
+* occurrence[x] only dateTime
+  * ^short = "Gibt das Datum an, an dem das Rezept ausgestellt werden soll"
+
+* authoredOn 1..1 MS
+  * ^short = "Datum der Erstellung des Anfrage"
+  * ^comment = "Wird initial angelegt und dann nicht mehr verändert."
+
+
+* requester 1..1 MS
+* requester only Reference(GEM_PR_MEDREQ_Organization)
+
+* performer MS
+* performer only Reference($KBV_PR_FOR_Practitioner)
+
+// TODO: Wenn reasonCode dann auch .note
+* reasonCode from GEM_VS_MEDREQ_MedicationRequest_Reason
+
+* reasonReference only Reference(GEM_PR_ERP_MEDREQ_RemainingMedication)
+
+* supportingInfo ^slicing.discriminator.type = #pattern
+* supportingInfo ^slicing.discriminator.path = "type"
+* supportingInfo ^slicing.rules = #open
```

Comment:
@simoneOnFhir das slicing gefällt mir noch nicht so. 
Hast du einen besseren Vorschlag wonach man das slicen kann?

Reply:
Setzt das Pattern auf reference (statt type) und path auf $this, dann seid ihr flexibler, falls ihr künftig mal noch einen slice mit einer logischen referenz braucht, wo ihr das pattern abweichend auf identifier.system setzen müsst.

Reply:
Wie genau? 
Meinst du hier das pattern setzen?
* supportingInfo[AusstellenderArzt].reference = ???

Ist dann nicht fix wie die Referenz aussehen soll?

Reply:
Äh ja, sorry, ich meinte das Pattern auf den *Datentyp* Reference setzen, also in dem Fall dann auf SupportingInfo das pattern "type=Practitioner" setzen.

Reply:
Also so wie es gerade ist?

* supportingInfo ^slicing.discriminator.type = #pattern
* supportingInfo ^slicing.discriminator.path = "type"
* supportingInfo ^slicing.rules = #open
* supportingInfo[AuslieferndeApotheke] only Reference(GEM_PR_MEDREQ_Organization)
* supportingInfo[AuslieferndeApotheke].type = "Organization

Die Konsequenz ist, dass nur eine Ressource von einem Typen eingebunden werden kann, korrekt?

Reply:
Ja genau. Wenn ihr künftig mal mehrere gleiche Ressourcentypen habt, müssten wir auf ein slicing by profile und path = resolve()  umstellen, aber da müsste ich erst mal umfangreiche Tests machen, bevor ich meine Hand dafür ins Feuer lege, dass das bei allen Validatoren richtig funktioniert. Für jetzt ist das o.g. Slicing das praktikabelste.